wpeverest/everest-forms
# Enhanced - Single Line Text/Paragraph fields limitation options

### All Submissions:






### Changes proposed in this Pull Request:



This PR aims to deliver limit option for Single Line Text and Paragraph fields.

### How to test the changes in this Pull Request:

1. Edit or add those supported fields.
2. Configure limit option to word/character.
3. View the form preview and confirm if it validates properly.

### Types of changes:




### Other information:




### Changelog entry

> Enhanced - Single Line Text/Paragraph fields limitation options (limit by character or word count).
